public/app/plugins/datasource/loki/configuration/ConfigEditor.test.tsx

```diff
@@ -24,7 +24,7 @@ describe('ConfigEditor', () => {
     const onChangeMock = jest.fn();
     const wrapper = mount(<ConfigEditor onOptionsChange={onChangeMock} options={createDefaultConfigOptions()} />);
     const inputWrapper = wrapper.find({ label: 'Maximum lines' }).find('input');
-    (inputWrapper.getDOMNode() as any).value = 42;
+    inputWrapper.getDOMNode<HTMLInputElement>().value = '42';
```

Comment:
Maybe consider using `assertInstanceOf` instead to get a bit more explicit safety from this? https://github.com/grafana/grafana/blob/8185b6fdf7b2f9bcef1f62dade4bb4087e23eba0/public/app/core/components/SharedPreferences/SharedPreferences.test.tsx

Reply:
I am curious about this as well. 🙂 I haven't seen using of `assertInstanceOf` before.

Reply:
Yeah - just personal flavor/busy work probably, but it's actually going to make sure the element is what you expect it to be, rather than essentially cast as `getDOMNode<HTMLInputElement>()` does. i very honestly meant _consider_ - if you don't feel the need, don't use it. just spreading awareness 😌

Reply:
Gotcha. I appreciate the suggestion, I will definitely keep it in mind! I think it should be enough as it is for this use case.